### ISSUE TITLE
Using Symbol instead of a named method for the operator overloading

### ISSUE THREAD

**Issue by Davilink** [kushal-likhi/operator-overloading-js#20]
Do you think it would be better to use Symbol instead of pure named method ? Method is suppose to be unique, so i guess it could be a good idea to use them, it like the iterator Symbol : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/iterator

**Comment by kushal-likhi** [comment]
Yes, looks like a good idea. Want to try that? i can work on it over the weekend.
